### PR TITLE
Flags resprite

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do

Resprite of flags
Some designes were pretty much random, some i shamelessly stole from internet with some adjustments, some are based on home planet description from lore.
Adds solgov and ussp flags to the game, one can be purchased in merchandise vendor and another can be ordered via hacked cargo console (along with syndie flag)
## Why It's Good For The Game

Old sprites bad (quite bad, head of generic vulp/taj/vox on flag is just ridiculous), new sprites good
## Images of changes

image

image
image
image
image

## Testing

image

## Changelog
:cl:
add: solgov and ussp flags, both obtainable in game
imageadd: New sprites for flags
imagedel:  Removed some useless "old" sprites from dmi
/:cl:

### CODE REVIEW

**Comment by datlo** [comment]
I like all of these except the Ian flag. I feel like the Ian flag should just be the ian sprite, it's funnier.

**Comment by Coolrune206** [comment]
The Clown flag will be in my nightmares, thank you.

File: code/modules/supply/supply_packs/pack_miscellaneous.dm [hunk 1]
```diff
@@ -360,6 +360,14 @@
 	containername = "crate"	//let's keep it subtle, eh?
 	contraband = TRUE
 
+/datum/supply_packs/misc/flags
+	name = "Unapproved flags Crate"
+	contains = list(/obj/item/flag/ussp,
+					/obj/item/flag/syndi,)
```
Comment (Sirryan2002) [inline code/modules/supply/supply_packs/pack_miscellaneous.dm:366]:
```suggestion
					/obj/item/flag/syndi)
```

**Review by Sirryan2002 — APPROVED** [review]
sprite diffs look good, same with code. One small nitpick

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**